taichi/util/short_name.cpp

```diff
@@ -0,0 +1,23 @@
+#include "short_name.h"
+#include "base64.h"
+
+TI_NAMESPACE_BEGIN
+namespace {
```

Comment:
Maybe remove this `namespace {` here?

Reply:
This is actually my advice. Anonymous namespace is like ‘static’, which hides your implementations details in one translation unit to avoid symbol conflicts.

> As a general rule of thumb in C++, please wrap such variables that are not exposed with an anonymous namespace. See https://stackoverflow.com/a/357427/12003165

Reply:
I see. I learned something today :-) Thanks.

Reply:
No, this is for ensuring not exposing local variables to global, as suggested by @k-ye:
> As a general rule of thumb in C++, please wrap such variables that are not exposed with an anonymous namespace. See https://stackoverflow.com/a/357427/12003165